projects/ngrx.io/content/guide/store/recipes/custom-elements.md

```diff
@@ -0,0 +1,88 @@
+# Using Store with Angular Elements
+## Creating Angular Elements
+The following recipe shows you how to manage the state of a counter, and how to select and display it within an Angular Elements and use the same within an Angular Project and a static page.
+## Recipe
+You can download  <live-example name="store-elements" downloadOnly>Angular Elements Project</live-example>.
+
+1. Create a new application by running the below command
+```sh
+ng new ngRxElementDemo - createApplication=false
+```
+
+### Creating Library
+
+1. Create a new Angular Library using below command
+```sh
+ng g library my-counter
+```
+2. Follow the instructions in the [Getting Started Guide](guide/store#installation) to create a new counter store complete with actions, reducer, and module.
+3. The only difference here is split the code for increment, decrement and reset in the sepearate component.
```

Comment:
```suggestion
3. The only difference here is to split the code for increment, decrement and reset into separate components.
```

Reply:
@santoshyadav198613 Still some typos, `reset into sepearate component.` should be `reset into separate components.`

Reply:
Done.

Reply:
@santoshyadav198613 Still a typo here: `sepearate components.` should be `separate components.`

Reply:
Done.